src/control/lib/control/pool.go

```diff
@@ -1056,7 +1086,7 @@ type (
 		Usage []*PoolTierUsage `json:"usage"`
 
 		// PoolRebuildStatus contains detailed information about the pool rebuild process.
```

Comment:
Comment still needs to be fixed
```suggestion
		// RebuildState contains detailed information about the pool rebuild process.
```